osgp/platform/osgp-domain-core/src/main/java/org/opensmartgridplatform/domain/core/entities/ScheduledTaskWithoutData.java

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright 2015 Smart Society Services B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.opensmartgridplatform.domain.core.entities;
+
+import java.io.Serializable;
+import java.sql.Timestamp;
+
+import javax.persistence.Entity;
+import javax.persistence.Table;
+
+import org.opensmartgridplatform.shared.infra.jms.DeviceMessageMetadata;
+
+/**
+ * Scheduled task without the actual data
+ */
+@Entity
+@Table(name = "scheduled_task")
+public class ScheduledTaskWithoutData extends AbstractScheduledTask {
+
+    private static final long serialVersionUID = 1L;
+
+    ScheduledTaskWithoutData() {
+        // Default empty constructor for Hibernate.
+    }
+
+    public ScheduledTaskWithoutData(final DeviceMessageMetadata deviceMessageMetadata, final String domain,
+            final String domainVersion, final Serializable messageData, final Timestamp scheduledTime) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "messageData". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AS1172)